README.md

```diff
@@ -42,10 +42,17 @@ Your `pom.xml` file may be modified to include the following dependency:
 <dependency>
   <groupId>com.iopipe</groupId>
   <artifactId>iopipe</artifactId>
-  <version>1.9.0</version>
+  <version>(replace with latest version of IOpipe)</version>
```

Comment:
I think a convention here would be something like LATEST_VERSION

Reply:
I know a convention that could be used to imply that it is something to be replaced by the user.